integration/data/cmd.py

```diff
@@ -101,6 +101,12 @@ def backup_inspect(url, backup):
     return json.loads(subprocess.check_output(cmd))
 
 
+def backup_list(url, name, dest):
```

Comment:
`backup_volume_list`

Reply:
this is the cmd that list backups for a backup volume, also we had a function "backup_volume_list"

Reply:
in fact, `backup_volume_list_backup`

Reply:
modified